src/coprocessor/error.rs

```diff
@@ -1,70 +1,97 @@
 // Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
 
-use std::error;
-use std::result;
-use std::time::Duration;
+use crate::storage;
 
-use kvproto::{errorpb, kvrpcpb};
-use tipb;
+#[derive(Fail, Debug)]
+pub enum Error {
+    #[fail(display = "Region error (will back off and retry) {:?}", _0)]
```

Comment:
I find it very strange to say "will back off and retry" here. First, the text can mean there is a back off and retry mechanism in TiKV side while there is not; second, the behaviour is up to the implement of client, it can either retry or just abort, so it gives a false promise.

It feels the same for "will clean up" at L10.